wp-admin/js/key-pair.js

```diff
@@ -0,0 +1,186 @@
+/* global keyPair, confirm, wp */
+( function( $, keyPair ) {
+	'use strict';
```

Comment:
I'm deferring review on this file because we'll likely want to refactor it to account for modern core JS standards later on.